dashboard/lib/feature_mode_manager.rb

```diff
@@ -60,16 +74,49 @@ class FeatureModeManager
         public_max_age: 8.hours.to_i,
       }
     },
+    fallback: {
+      gatekeeper_general_settings: {
+        puzzle_rating: false,
+        hint_view_request: true,
+        postMilestone: true,
+        postFailedRunMilestone: true,
+        shareEnabled: true,
+        slogging: false
+      },
+      gatekeeper_hoc_tutorial_settings: {
+        postMilestone: true,
+        postFailedRunMilestone: false,
+        shareEnabled: true,
+      },
+      gatekeeper_csf_tutorial_settings: {
+        postMilestone: true,
+        postFailedRunMilestone: true,
+        shareEnabled: true,
+      },
+      dcdo_settings: {
+        hoc_activity_sample_weight: 10,
+        hoc_learn_activity_sample_weight: 0,
+        public_proxy_max_age: 4.hours.to_i,
+        public_max_age: 8.hours.to_i,
+      }
+    },
```

Comment:
Would it be possible to make the settings from the 'higher'-level mode be inherited by the 'lower'-level one, so this would be easier to read? I worry that with duplicating the large list of configurations across each mode, we might overlook something important in the copy/paste process or forget to update one or the other in sync.

(e.g.: is `hoc_learn_activity_sample_weight: 0` in this new mode intentionally matching the `emergency` setting, or an error?)